docs/sphinx/user_guide/run.rst

```diff
@@ -130,8 +135,19 @@ is not the name of a kernel in the Suite, while DAXPY is the name of a kernel.
 Running with MPI
 ==================
 
-Running the Suite with MPI is just like running any other MPI application.
-For example, issuing the following command on a machine with slurm scheduling::
+The Suite can be configured and compiled to run in a distributed memory
+parallel mode using MPI. Running the Suite on multiple MPI ranks will execute
+the same code for each kernel on each rank with minimal synchronization points
+to gather execution timing data from all ranks. This capability is provided so
+that individual kernel performance more closely aligns with how such kernels 
+would perform in a real application. For example, compute node memory bandwidth
+impact on performance may be different when running on many core system using
+OpenMP multithreading to exercise all cores than when each core is mapped to
+an MPI rank.
```

Comment:
```suggestion
would perform in a real application. For example, compute node memory bandwidth
may be different when running on many core system using
OpenMP multithreading to exercise all cores than when each core is mapped to
an MPI rank.
```

Reply:
Fixed.  @MrBurmark if you have no other suggestions, I will merge when checks are green.